sound/soc/sof/intel/skl.c

```diff
@@ -92,7 +92,7 @@ const struct snd_sof_dsp_ops sof_skl_ops = {
 EXPORT_SYMBOL(sof_skl_ops);
 
 const struct sof_intel_dsp_desc skl_chip_info = {
-	/* Apollolake */
+	/* Skylake */
```

Comment:
this is really not related to the timeout, you need to provide patches that match the commit message

Reply:
@plbossart OK, will sub another PR